transport/rif_comms/node.py

```diff
@@ -240,7 +240,14 @@ def send_message(self, payload: str, recipient: Address):
         """
         Send text message through the RIF Comms client.
         """
-        self._comms_client.subscribe_to(recipient)
+        self.log.info(
+            "RIF Comms sending message", message_payload=payload.replace("\n", "\\n"), recipient=pex(recipient)
```

Comment:
per my other comment:
```suggestion
            "sending message", message_payload=payload.replace("\n", "\\n"), transport="rif_comms", recipient=pex(recipient)
```